mono/metadata/ee-internals.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+#include <config.h>
+#include <mono/metadata/metadata.h>
+#include <mono/metadata/object.h>
+#include <mono/utils/mono-compiler.h>
+#include <mono/utils/mono-error.h>
+#include <mono/eglib/glib.h>
+
+#ifndef __MONO_EE_INTERNALS_H__
+#define __MONO_EE_INTERNALS_H__
+
+typedef struct _MonoInterpStackIter MonoInterpStackIter;
+
+/* Needed for stack allocation */
+struct _MonoInterpStackIter {
+	gpointer dummy [8];
+};
+
+typedef gpointer MonoInterpFrameHandle;
+
+struct _MonoEECallbacks {
+	gpointer (*create_method_pointer) (MonoMethod *method, MonoError *error);
+	MonoObject* (*runtime_invoke) (MonoMethod *method, void *obj, void **params, MonoObject **exc, MonoError *error);
+	void (*init_delegate) (MonoDelegate *del);
+#ifndef DISABLE_REMOTING
```

Comment:
Since this function pointer is optional in the structure, it must be part of the version number/string.

Reply:
good point!

Reply:
Or the field should not be optional.